doc-content/enterprise-only/admin-and-config/ldap-login-domain-proc.adoc

```diff
@@ -1,9 +1,9 @@
 [id='ldap-login-domain-proc']
 = Define the LDAP login domain
 
-When you are setting up {PRODUCT} to use LDAP for authentication and authorization, define the LDAP login domain. This is because the Git SSH authentication may be using another security domain, in which case you may face authentication failure. 
+When you are setting up {PRODUCT} to use LDAP for authentication and authorization, define the LDAP login domain. This is because the Git SSH authentication may be using another security domain, in which case you may face authentication failure.
```

Comment:
"... define the LDAP logon domain as the the Git SSH authentication may use another security domain.

Reply:
Adopted and changed "as" to "because"